.github/workflows/release.yaml

```diff
@@ -0,0 +1,34 @@
+# releases packages once a version has been tagged
+name: Release App
+
+on:
+  release:
+    types:
+      - created
+
+jobs:
+  goreleaser:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Set up Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: 1.16.x
```

Comment:
we got `1.16.x` here but `1.17.x` in https://github.com/adfinis-sygroup/mopsos/pull/1#diff-733fa1a1798b2a08a08b8c4f2ae7c7a0d7bfd9e0eb6e8796f60ef5428ba3a335R27?

Reply:
```suggestion
          go-version: 1.17.x
```